### DISCUSSION TITLE
can't I select it in the build

### DISCUSSION THREAD

**Discussion by Nilix5** [codemagic-ci-cd/codemagic-docs#1003] [Q&A]
I have an app in codemagic that I want to import into IOS. But I can't create because I can't select the branch(main). But the code is available in github and the path is correct, why can't I select it in the build?
i get -> Cannot access repository https://api.github.com/repos/xxxx/xxxx

**Comment by K-Codemagic** [comment]
Here are some tips on what to check if you can’t see your repository listed in Codemagic or it’s shown as unavailable.

- If you chose to grant access to select repositories only as opposed to all repositories when setting up the GitHub App, make sure you have also granted access to the repository in question. Click Manage integration > Configure in a team or user integrations to configure repository access settings.
- Your personal Codemagic account and each team have separate integrations. Make sure you have connected the GitHub App integration to the right team or in your user settings.
- If you build apps from different GitHub accounts or organizations, make sure that you have installed the GitHub App on the account that has the repository available. Click Manage integration > Configure to see which accounts have been connected. You can also visit https://github.com/settings/installations for personal accounts or github(dot)com/organizations/your-organization/settings/installations for organization accounts to check whether Codemagic CI/CD is listed both under installed and authorized GitHub Apps.
- Private GitHub repositories can be shared with Codemagic only when the GitHub App is installed on the account that owns the repository. If you are a collaborator to a private repository, please ask the repository owner to install Codemagic GitHub App on GitHub: https://github.com/apps/codemagic-ci-cd

Let me know incase you still face this issue.

**Comment by Nilix5** [comment]
Thanks for youre answer.
it is a private GitHub repositorie and codemagic app is installed with read write access for all repositories.

Everything has worked before, after 6 months I wanted to make changes and it no longer works.

i also install new Codemagic CI/CD but no changes.
Also when i want ass new app then it shows me Codemagic CI/CD but the connection circle is turning uninterrupted.

When i go in codemagic to user settings -> Integrations -> GitHub -> GitHub App account (Nilix5) has been connected

**Reply by K-Codemagic** [reply]
Can you share the app id

**Comment by Nilix5** [comment]
Hello

 

App id https://github.com/Nilix5/Studio-Manager-V2

 

 

Mit freundlichen Grüßen

Sincerely

Frank Schneider

 

 

 

Von: K-Codemagic ***@***.*** 
Gesendet: Freitag, 30. September 2022 14:15
An: codemagic-ci-cd/codemagic-docs ***@***.***>
Cc: Nilix5 ***@***.***>; Author ***@***.***>
Betreff: Re: [codemagic-ci-cd/codemagic-docs] can't I select it in the build (Discussion #1003)

 

Can you share the app id

**Comment by Nilix5** [comment]
Now it works

**Reply by K-Codemagic** [reply]
Great
